manifests/nacos/injection-template.yaml

```diff
@@ -0,0 +1,77 @@
+metadata:
```

Comment:
We don't need this.

What we want is not use nacos and service mesh at the same time.